rm: security: improve boot software updates article for zynqmp (#414)
foundriesio/docs: source/reference-manual/security/boot-software-updates-zynqmp.rst

* In **QSPI** boot case multi_boot value represents multiple of 32 KB (as
  boot image should be placed on 32 KB boundary), so to force BootROM to
  boot image from **0x60000** offset on QSPI, **CSU_MULTI_BOOT** register

>   value have to be set to **0xC** (hex) or **12** (dec), because
>   `0x60000 = 0x8000 * 0xC`, where **0x8000** is 32KB boundary (in hex) and
>   **0xC** is multiboot value (also in hex).

Reviewer: ```suggestion
  value needs to be set to **0xC** (or **12** in dec) because
  `0x60000 = 0x8000 * 0xC`, where **0x8000** is the 32KB boundary and
  **0xC** is the multiboot value.
```
the 0x prefix denotes the value as being in hex, so I feel like we do not need to call it out further.

    Trying to boot from SPI
    SPL: Booting next image from 0xaa0000 SPI offset

> From output you can see that after setting secondary boot (`multi_boot 12` or
> `multi_boot 0xc`, both decimal and hexademical values are supported)

Reviewer: ```suggestion
From the output you can see that after setting the secondary boot (`multi_boot 12` or
`multi_boot 0xc` as both dec and hex values are supported)
```
